Gems/Atom/Feature/Common/Code/Source/CoreLights/ProjectedShadowmapsPass.cpp

```diff
@@ -212,22 +215,48 @@ namespace AZ
             m_updateChildren = true;
             UpdateChildren();
 
-            // [GFX TODO][ATOM-2470] stop caring about attachment
             RPI::Ptr<RPI::PassAttachment> attachment = m_ownedAttachments.front();
             if (!attachment)
             {
                 AZ_Assert(false, "[ProjectedShadowmapsPass %s] Cannot find shadowmap image attachment.", GetPathName().GetCStr());
                 return;
             }
-            AZ_Assert(attachment->m_descriptor.m_type == RHI::AttachmentType::Image, "[ProjectedShadowmapsPass %s] requires an image attachment", GetPathName().GetCStr());
-
-            RPI::PassAttachmentBinding& binding = GetOutputBinding(0);
-            binding.SetAttachment(attachment);
+            if (attachment->m_descriptor.m_type != RHI::AttachmentType::Image)
+            {
+                AZ_Assert(false, "[ProjectedShadowmapsPass %s] requires an image attachment", GetPathName().GetCStr());
+                return;
+            }
 
             RHI::ImageDescriptor& imageDescriptor = attachment->m_descriptor.m_image;
             const uint32_t shadowmapWidth = static_cast<uint32_t>(m_atlas.GetBaseShadowmapSize());
             imageDescriptor.m_size = RHI::Size(shadowmapWidth, shadowmapWidth, 1);
+            imageDescriptor.m_format = RHI::Format::D32_FLOAT;
```

Comment:
I guess the pass system wont read the format from .pass file if the attachment is imported

Reply:
I decided to just move that to the cpp since most of the rest of the descriptor is defined there anyway.